var/spack/repos/builtin/packages/python/package.py

```diff
@@ -197,6 +197,7 @@ class Python(AutotoolsPackage):
     patch('python-3.7.4+-distutils-C++.patch', when='@3.7.4:')
 
     patch('tkinter.patch', when='@:2.8,3.3:3.7 platform=darwin')
+    patch('tkinter-3.8.patch', when='@3.8: ~tkinter')
```

Comment:
Can you add a brief comment on top of the directive to explain what the patch does, why it is needed and where it comes from?

Reply:
Done. I wrote it myself so I'm not sure what to put for attribution?

Reply:
> I wrote it myself so I'm not sure what to put for attribution?

Now that's on record on Github for anybody that needs it in the future :laughing: LGTM, so merging